inc/saiacl.h

```diff
@@ -816,7 +816,7 @@ typedef enum _sai_acl_table_attr_t
     /**
      * @brief Range type defined
      *
-     * @type sai_acl_range_type_t
+     * @type sai_s32_list_t sai_acl_range_type_t
```

Comment:
It is no different than any other match field, which are all create only - 
CREATE_ONLY, match fields cannot be changed after the table is created
When you create the table, you need to specify all the possible match fields in the key
If you then try to create an entry with different keys, you should get an error